addon/components/polaris-text-field.js

```diff
@@ -348,7 +400,15 @@ export default Component.extend(
      */
     onBlur() {},
 
+    /**
+     * @private
+     */
+    focus: false,
+
+    dataTestTextField: 'text-field',
```

Comment:
Any reason some of these files are formatted this way and other are using snakecase with `true` values? If not we should decide on sticking to one format.

Reply:
yes
I think I mentioned this before....the reason is that for tag-less components, passing `data-test-*` format will cause ember-test-selectors to throw a warning (since you can't add something if we don't have an element for the component)....so instead we pass camelCased format and bind it to the first element as a test seletor in the tag-less component template

let me know if that doesn't makes sense

Reply:
Ahh yes I forgot. My bad 😞

Reply:
haha, no harm in asking 😉